README.md

```diff
@@ -1,45 +1,45 @@
+* **MSFT Employees**: If you are a PFE, on the support team, program manager, or developer from the BizTalk or HIS product teams, then it's your job to contribute to or author technical articles. If you are making substantial changes to an existing article, adding or changing images, or contributing a new article, then we suggest to fork this repository, install Git Bash, a markdown editor ([VS Code](https://code.visualstudio.com/download) is preferred), and learn some git commands. The [internal contributor's guide](https://review.docs.microsoft.com/help/contribute/) provides the details to get started.
 
 ## Repo rules
-When you update a topic, please: 
+When you update a topic:
 
 1. Make your changes to the **Live** branch. There may be other branches, but Live is the branch that is published to `docs.microsoft.com/biztalk`.
 
-2. Change **ms.date** to the date you update the topic. Here's an example: 
+2. Change **ms.date** to the date you update the topic. Here's an example:
 
-    ```
+    ```yml
     ms.date: "11/28/2017"
     ```
 
-3. **Description** may be used for SEO. If it's missing a description, please add one. Here's an example: 
+3. **Description** may be used for SEO. If it's missing a description, please add one. Here's an example:
 
-    ```
-    title: Configure using Basic or Custom configuration | Microsoft Docs
+    ```yml
+    title: Configure using Basic or Custom configuration
     description: Steps to do a basic or custom configuration of BizTalk Server, and learn what happens with each configuration
     ```
 
-4. Want some guidance or help for your pull request? Tag `@mandiohlinger` in your pull request, and I'll add my notes to the comments. 
+4. Want some guidance or help for your pull request? Tag `@mandiohlinger` in your pull request, and I'll add my notes to the comments.
 
 ## BizTalk repository structure
 
 ### \adapters-and-accelerators
 Contains all the installation and conceptual documentation for the BizTalk Adapter Pack (BAP), HL7, RosettaNet, SWIFT, FileAct/Interact, and the WCF LOB Adapter SDK. It does not include the built-in or native adapters, such as FTP, or WCF-WebHTTP. These are included in *\core*.
 
 ### \core
-Contains all the conceptual documentation, including getting started, tutorials, performance, the built-in adapters, B2B, and BAM. These topics typically apply to multiple BizTalk versions. 
+Contains all the conceptual documentation, including getting started, tutorials, performance, the built-in adapters, B2B, and BAM. These topics typically apply to multiple BizTalk versions.
 
 ### \dev-centers
-Contains all the stand-alone topics at [https://docs.microsoft.com/biztalk](https://docs.microsoft.com/biztalk), including the community-written blogs.
+Contains all the stand-alone topics in [Microsoft BizTalk Server documentation](https://docs.microsoft.com/biztalk)(, including the community-written blogs.
```

Comment:
An extra "(" was added that needs to be removed.

Reply:
Fixed